content/en/Platform Deep Dive/Collaboration/user-roles.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "User Roles and Permissions"
+linkTitle: "User Roles"
+weight: 30
+aliases: /platform-deep-dive/collaboration/user-roles/
+description: >
+  Learn about the user roles and associated permissions.
+---
+
+{{% pageinfo %}}
+Depending on your role, you can collaborate at the pentest or organization level.
+{{% /pageinfo %}}
+
+| [Pentest Roles](#pentest-roles) | [Organization Roles](#organization-roles) |
+|---|---|
+| [Pentest Team Member](#pentest-team-member):<ul><li>Collaborates on a specific pentest.</li><li>Has no access to organization users and settings,<br>unless the user is also an Organization Owner or Member.</li></ul>Cobalt [Pentester](#pentester) and [Pentest Lead](#pentest-lead):<ul><li>Work on pentests they're assigned to.</li></ul> | [Organization Owner](#organization-owner):<ul><li>Can create and edit assets and pentests.</li><li>Manages organization users and settings.</li></ul>[Organization Member](#organization-member):<ul><li>Can create and edit assets and pentests.</li><li>Can view organization users and settings.</li></ul> |
```

Comment:
Org owners are only able to edit pentests if they are a team member on that pentest. I would remove "and pentests". https://www.loom.com/share/31d15d4a9edf4acfb07c8716c43d3b8f